wolfcrypt/src/tfm.c

```diff
@@ -5808,7 +5824,12 @@ int mp_toradix (mp_int *a, char *str, int radix)
         return FP_MEM;
 #endif
 
-    /* init a copy of the input */
+/* Appease compiler for configurations known to need it.
+**   CONFIG_COMPILER_OPTIMIZATION_PERF: Espressif -O2 */
+#if defined(CONFIG_COMPILER_OPTIMIZATION_PERF)
+    fp_init(t); /* Init x to appease some compilers before call to get copy. */
```

Comment:
Same here: I'd prefer to see a PR where these are always initialized.

Reply:
Agreed. See latest code changes.